test/verify/check-packagekit

```diff
@@ -44,7 +44,7 @@ class NoSubManCase(PackageCase):
 
         # Disable Subscription Manager on RHEL for these tests; subscriptions are tested in a separate class
         # On other OSes (Fedora/CentOS) we expect sub-man to be disabled in yum, so it should not get in the way there
-        if self.machine.image.startswith("rhel") or self.machine.image.startswith("centos-8"):
+        if self.machine.image in ["rhel-8-2", "rhel-8-2-distropkg", "centos-8-stream"]:
```

Comment:
Fun, earlier this morning I saw exactly that failure in an 8.3 downstream test. Is rhsmd just not a thing any more?

Reply:
> Is rhsmd just not a thing any more?

indeed, it is present only when you install `subscription-manager-gui`.